### DISCUSSION TITLE
Question: how to match array keys?

### DISCUSSION THREAD

**Discussion by hanayashiki** [TanStack/query#2750] [Q&A]
https://react-query.tanstack.com/guides/filters#query-filters

In the tutorial I only see how to match string keys. Is there any official ways to match array keys (except for using predicator)?

Thanks in advance.

**Comment by TkDodo** [comment]
unless you pass `exact` to a query filter, the keys match fuzzily. I have some examples in my blog post about query keys: https://tkdodo.eu/blog/effective-react-query-keys
